test/jdk/java/net/httpclient/ProxyServer.java

```diff
@@ -195,6 +195,9 @@ class Connection {
         volatile InputStream clientIn, serverIn;
         volatile OutputStream clientOut, serverOut;
 
+        volatile boolean proxyInClosed;
+        volatile boolean proxyOutClosed;
```

Comment:
nit: these variables don't need to be volatile; they are only accessed in a synchronized block

Reply:
Good point